### PR TITLE
Added XMLRPC support for stunnel

### PR DESCRIPTION
**Pull request by infraweavers**
Following the same pattern as: https://github.com/pfsense/FreeBSD-ports/pull/967/files

### CODE REVIEW

File: security/pfSense-pkg-stunnel/files/usr/local/pkg/stunnel.inc [hunk 1]
```diff
@@ -147,4 +149,47 @@ function stunnel_deinstall() {
...
+
+	$synctimeout = intval($synctimeout);
+	$rpc_client = new pfsense_xmlrpc_client();
+	$rpc_client->setConnectionData($sync_to_ip, $port, $username, $password, $protocol);
+	$resp = $rpc_client->xmlrpc_method('merge_installedpackages_section', $xml, $synctimeout);
+	$resp = $rpc_client->xmlrpc_exec_php($remote_stunnel_config_apply, $synctimeout);
+}
+
 ?>
+
```
Comment (jim-p) [inline security/pfSense-pkg-stunnel/files/usr/local/pkg/stunnel.inc:195]: Remove the empty line(s) here at the end

**Review by jim-p — CHANGES_REQUESTED** [review]
As a part of this pull request, you must increase PORTVERSION or add a PORTREVISION line in the package Makefile. Without this version increase, the package will not be rebuilt to include the new change.

See https://docs.netgate.com/pfsense/en/latest/references/developer-style-guide.html#ports-packages-specific-rules

**Comment by rbgarga** [comment]
Manually merged to get commits squashed.  Thanks!

**Comment by infraweavers** [comment]
@rbgarga @jim-p thanks, we'll squash all our commits into 1 for future PRs

**Pull request closed without merging**